sql/2021/pwa/popular_pwa_libraries_helper.sql

```diff
@@ -0,0 +1,54 @@
+#standardSQL
+# Use this sql to find popular library imports for popular_pwa_libraries.sql
+# And also other importscripts used in service workers
+CREATE TEMPORARY FUNCTION getSWLibraries(importScriptsInfo STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS '''
+try {
+  /* 'importScriptsInfo' returns an array of libraries that might import other libraries
+     The final array of libraries comes from the combination of both */
+  var ObjKeys = Object.keys(JSON.parse(importScriptsInfo));
+  var ObjValues = Object.values(JSON.parse(importScriptsInfo));
+  if (typeof ObjKeys == 'string') {
+    ObjKeys = [ObjKeys];
+  }
+  if (typeof ObjValues == 'string') {
+    ObjValues = [ObjValues];
+  }
```

Comment:
See previous comment by Rick. These two branches aren't needed since `Object.keys` and `Object.values` always returns an array
```suggestion
```